README.rst

```diff
@@ -103,24 +103,161 @@ Available Features
 
 ------------
 
-Install Dependencies
---------------------
+Installation instructions
+-------------------------
+
```

Comment:
I think we need a "Deploy it in production" section like the one we have in OpenWISP Controller: https://github.com/openwisp/openwisp-controller#deploy-it-in-production.